zerver/tests/test_push_notifications.py

```diff
@@ -76,6 +77,7 @@
     get_display_recipient,
     get_realm,
     get_stream,
+    get_user_profile_by_id,
```

Comment:
This is unused in this commit

Reply:
@alexmv Moved to the correct ` push_notifications: Use uuids to identify users to the bouncer.` commit